webapp/components/notifications/Notification/Notification.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <ds-space :class="[{ read: notification.read }, notification]" margin-bottom="x-small">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-19T14:42:56Z" title="Monday, August 19th 2019, 4:42:56 pm +02:00">Aug 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,89 @@
+<template>
+  <ds-space :class="{ notification: true, read: notification.read }" margin-bottom="x-small">
```


I think you could use `:class="[{ read: notification.read }, notification]"`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-20T05:37:05Z" title="Tuesday, August 20th 2019, 7:37:05 am +02:00">Aug 20, 2019</time>_

---


That’s okay with me, in case @mattwr18 likes to do it, because he will be responsible for today … @alina-beck